.github/vale/config/vocabularies/Backstage/accept.txt

```diff
@@ -24,6 +24,7 @@ Avro
 backend's
 backported
 backporting
+[Bb]ackstage
```

Comment:
hm this hasn't been needed before now

Reply:
This was related to vale version I had installed. Removing,